lib/spack/spack/environment/__init__.py

```diff
@@ -16,11 +16,12 @@
 The high-level format of a Spack lockfile hasn't changed much between versions, but the
 contents have.  Lockfiles are JSON-formatted and their top-level sections are:
 
-  1. ``_meta`` (object): this contains deatails about the file format, including:
+  1. ``_meta`` (object): this contains details about the file format, including:
       * ``file-type``: always ``"spack-lockfile"``
       * ``lockfile-version``: an integer representing the lockfile format version
       * ``specfile-version``: an integer representing the spec format version (since
         ``v0.17``)
+      * ``spack-commit``: the Spack git commit (since ``v0.19``)
```

Comment:
This doesn't belong under `_meta` -- it's a top level field.  `_meta` is really about the format of *this* file.

Reply:
Okay.  I didn't realize that.  Thanks!